discovery-provider/src/tasks/users.py

```diff
@@ -211,8 +213,37 @@ def get_metadata_overrides_from_ipfs(session, update_task, user_record):
         if ipld_blacklist_entry:
             return None
 
+        # Manually peer with user creator nodes
+        update_ipfs_peers_from_user_endpoint(
+            update_task,
+            user_record.creator_node_endpoint)
+
         user_metadata = update_task.ipfs_client.get_metadata(
             user_record.metadata_multihash,
             user_metadata_format)
 
     return user_metadata
+
+def get_ipfs_info_from_cnode_endpoint(url):
+    id_url = urljoin(url, 'ipfs_peer_info')
+    resp = requests.get(id_url)
```

Comment:
@hareeshnagaraj is there going to be a timeout here? what about if a creator node doesn't exist anymore or fails to respond

Reply:
this is a great comment...there should absolutely be a timeout if this happens